plugins/emoji/plugin.js

```diff
@@ -629,6 +629,16 @@
 						data = arrTools.filter( emojiList, function( item ) {
 							// Comparing lowercased strings, because emoji should be case insensitive (#2167).
 							return item.id.toLowerCase().indexOf( emojiName ) !== -1;
+						} ).sort( function( a, b ) {
+							var aIndex = a.id.substr( 1 ).indexOf( emojiName ),
+								bIndex = b.id.substr( 1 ).indexOf( emojiName );
+
+							// True when left and right side are oposite.
+							if ( aIndex === 0 ^ bIndex === 0 ) {
```

Comment:
This could be simplified down to `Boolean( aIndex ) != Boolean( bIndex )`. Reader will not have to google for bit operators and refresh the memory. 🙂

Reply:
I've move even further and set boolean value of variable instead of preserving its index. Index is not used anywhere. Now sort condition should looks much more neat.